designdocs/Deterministic wallets.txt

```diff
@@ -81,10 +87,8 @@ it's less convenient than an integrated solution would be.
 Chain structure
 ---------------
 
-We follow the suggested chain structure outlined in BIP32 in which there is the notion of a top-level account, though
-in bitcoinj this will always be zero as it won't be exposed in the API for now. Under the account are two keys, one
-for receiving of funds (i.e. exposed to the use) and one which is used for "internal purposes", typically change keys
-though it may also be used for things like micropayment channels and, in future, de/refragmentation.
```

Comment:
Shouldn't this behavior still be documented somewhere?

Reply:
Somehow I thought it's better if people read up the mentioned BIPs (32/43/44/84/86). I think the idea about internal and external addresses is covered there very well.

Also I feel talking too much about features that _could_ be implemented, but after all these years haven't been, distracts the reader from what bitcoinj can actually do.